src/main.rs

```diff
@@ -422,8 +417,9 @@ fn real_main() -> Result<(), miette::Report> {
         Some(FetchImports(sub_args)) => cmd_fetch_imports(out, &cfg, sub_args),
         Some(RegenerateUnaudited(sub_args)) => cmd_regenerate_unaudited(out, &cfg, sub_args),
         Some(DumpGraph(sub_args)) => cmd_dump_graph(out, &cfg, sub_args),
-        // Need to be non-exhaustive because freestanding commands were handled earlier
-        _ => unreachable!("did you add a new command and forget to implement it?"),
+        Some(Inspect(sub_args)) => cmd_inspect(out, &cfg, sub_args),
+        Some(Diff(sub_args)) => cmd_diff(out, &cfg, sub_args),
```

Comment:
Are we ok with regressing #29? (not regressing it is a frustrating problem to solve...)

Reply:
I would prefer not to regress it. How much work is it to handle?

Reply:
Currently we have a hard distinction between "commands which can access the store and therefore require one" and "commands which can't access the store" with a type-level distinction on the kind of "Config" that gets passed in. This would require adding the ability to detect whether the store exists ahead of time and optionally enabling features based on that.

Reply:
(Which would completely change the way main/config is handled)

Reply:
I think I'm actually OK with regressing it for now, which will also ensure people don't unintentionally miss our guidance flow.